package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "3.4.9",
+  "version": "3.4.10",
```

Comment:
The reason that the CI checks are still failing is that you updated the "vue-test-utils-helpers" import in the package.json file but you did not run `npm i` afterwards (and/or you did not commit the updated package-lock.json file).

The build server uses THIS file to determine what packages to install (and what version). This way, the build server will use exactly the same packages to build the app that you used locally.